### PR TITLE
#80 add component patch template

### LINKED ISSUE #80: Add component patch template

### PR DESCRIPTION
**Pull request by meiserloh**
Resolves #80

### CODE REVIEW

File: k8s/component-patch-tpl.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+values:
+  images:
+    k8sCesSetup: cloudogu/k8s-ces-setup:0.19.1
+    kubectl: bitnami/kubectl:1.27.4
+    etcdClient: bitnami/etcd:3.5.2-debian-10-r0
+
+patches:
+  values.yaml:
+    k8s_ces_setup_image: "{{ registryFrom .images.k8sCesSetup }}/{{ repositoryFrom .images.k8sCesSetup }}:{{ tagFrom .images.k8sCesSetup }}"
```
Comment (nhinze23) [inline k8s/component-patch-tpl.yaml:11]:
```suggestion
    k8s_ces_setup_image: "{{ .images.k8sCesSetup }}"
```

File: k8s/component-patch-tpl.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    kubectl_image: "{{ registryFrom .images.kubectl }}/{{ repositoryFrom .images.kubectl }}:{{ tagFrom .images.kubectl }}"
```
Comment (nhinze23) [inline k8s/component-patch-tpl.yaml:12]:
```suggestion
    kubectl_image: "{{ .images.kubectl }}"
```

**Comment by nhinze23** [comment]
Missing changelog

**Review by kahoona77 — APPROVED** [review]
LGTM

**Pull request merged**